src/cpu-template-helper/src/template/dump/aarch64.rs

```diff
@@ -34,9 +37,24 @@ pub fn config_to_template(cpu_config: &CpuConfiguration) -> CustomCpuTemplate {
     }
 }
 
+// List of register IDs excluded from the CPU configuration dump.
+static REG_EXCLUSION_LIST: &[u64] = &[
+    // SYS_CNTV_CVAL_EL0 and SYS_CNTPCT_EL0 are timer registers and depend on the elapsed time.
+    // This type of registers are not useful as guest CPU config dump.
+    SYS_CNTV_CVAL_EL0,
+    SYS_CNTPCT_EL0,
+    // Program counter (PC) value is determined by the given kernel image. It should not be
+    // overwritten by a custom CPU template and does not need to be tracked in a fingerprint file.
+    PC,
+];
```

Comment:
It is better to define this as `const`. If you want to use it as a slice later you do do `&REG_EXCLUSION_LIST`.
```suggestion
// List of register IDs excluded from the CPU configuration dump.
const REG_EXCLUSION_LIST: [u64; 3] = [
    // SYS_CNTV_CVAL_EL0 and SYS_CNTPCT_EL0 are timer registers and depend on the elapsed time.
    // This type of registers are not useful as guest CPU config dump.
    SYS_CNTV_CVAL_EL0,
    SYS_CNTPCT_EL0,
    // Program counter (PC) value is determined by the given kernel image. It should not be
    // overwritten by a custom CPU template and does not need to be tracked in a fingerprint file.
    PC,
];
```

Reply:
fixed

Reply:
I opened https://github.com/rust-lang/rust-clippy/issues/12518 to check this.